apps/web/src/components/Profile/Followers.tsx

```diff
@@ -94,6 +94,9 @@ const Followers: FC<FollowersProps> = ({ profile }) => {
                   showFollow={
                     currentProfile?.id !== follower?.wallet?.defaultProfile?.id
                   }
+                  showUnfollow={
+                    currentProfile?.id !== follower?.wallet?.defaultProfile?.id
+                  }
```

Comment:
I guess we can make a common props for both 🙇🏼

Reply:
good point! i'll update it

Reply:
@bigint Just want to double check. If we replace showFollow and showUnfollow, with a common prop, (let's say showFollowUnfollow), then it would also change the behavior a bit in other places where UserProfile is used. We would show an Unfollow button in more places.

Example: I am following TeaParty. With this change I would now see an Unfollow in the circle in this screenshot below. Is this desired? I think it would be an improvement.

![Screenshot_2023-09-02_11-56-37](https://github.com/lensterxyz/lenster/assets/667227/ea6b90e6-ec16-4409-8310-5d3ea0158a54)

Reply:
These are for current account only right? how it will hide for other account which is not owned by you 🤔

Reply:
A bit hard to explain, but i'll try  🙂
First, yes, this is about when currentProfile sees a another persons profile in in a list or somewhere else at the site.
Also, this is just a small change that I wanted to mention, about the consequence of using a common boolean prop for showFollow and showUnfollow. No big change, but a slight change in behavior, as follows:

If we replace, showFollow and showUnfollow with a common prop, "showFollowUnfollow", when we will **show** the unfollow button in more places, for example in the circle in the screenshot. Because before, in many uses of UserProfile, we only passed showFollow as a prop (and not showUnfollow). But if we replace these with a common boolean, the logic will be the same for follow and unfollow, and thus, we will show **Unfollow** button on all places where we only showed **Follow**.

Reply:
Addition: I hope this answers your question. If not, maybe we should instead try to write test cases first, before implementing the logic? Then the expected behavior will be clear from the beginning.

Also one need to keep in mind that there are **three** parties involved in the logic. A: The current account, B: The account who owns the Following/Followers list, and C. The account of each profile displayed in the list. And in each case A, B and C may or may not be the same account.

Reply:
Ahh i got you what you mean 🙌🏼  makes sense 🚀